controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1321,52 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * caller function for client_get_client_list_caller
+ *
+ * @param [in] numOfArgs Num of received arguments
+ *
```

Comment:
Please remove spaces between `@param` and `@return`.
Also missing dot at the end of the sentence.
Also (2), Start with a capital letter.
All are repeated several times on the file.

Reply:
Fixed